CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ SPDX-License-Identifier: BSD-3-Clause
 
 # w3c/vc-di-ecdsa-test-suite  ChangeLog
 
+## 2.3.0 -
+
+### Added
+- A new `credentials` section in `runner.json` that allows configuring static test vectors.
+- A new script that clones the `vc-di-ecdsa` repo and `TestVectors` into the project.
+- A new `mocks` dir that contains static test vectors and respective pointers.
+- A new `issuerName` config property for configuring the issuer used to create test data.
+- A new `holderName` config property for configuring the holder used to create sd test data.
+
+### Changed
+- The env variables for specifying the issuer for the rdfc & sd tests are now separate.
```

Comment:
We should be consistent with abbreviation capitalization. So...
```suggestion
- The env variables for specifying the issuer for the RDFC & SD tests are now separate.
```

Reply:
True in many other places in this PR. If you need to wait to keep the rest of this moving, that's fine also.